examples/instrumentation-quickstart/build.gradle

```diff
@@ -25,6 +25,10 @@ group = 'com.example'
 version = '0.0.1-SNAPSHOT'
 description = 'OpenTelemetry instrumentation quickstart using Spring Boot'
 
+java {
+	sourceCompatibility = '17'
+}
+
```

Comment:
Since the minimum compatibility is different from other examples, could you mention the minimum compatibility in the README for this example ?